docs/docsPrototype.md

```diff
@@ -0,0 +1,165 @@
+# Documentation
+
+## Glossary
+- Agent - single application instance, is able to compute one subproblem (task) at a time
+- Computational problem - problem solvable with Codeine. It should be divisible into a finite amount of subproblems (tasks) which can be solved independent of each other
+- Computational network (usually referred to as "network") - a network of agents communicating with each other, who together solve one computational problem
+- Task - a single subproblem of the computational problem
+
+## Requirements
+- Project should implement peer-to-peer networking on LAN
+- We should assume that about 5 agents at once can work on our computational problem
+- Every agent should have exactly the same application
+- After application launch, agent should automatically connect with all other agents in the network
```

Comment:
More like "should automatically _attempt_ to discover other agents present on LAN"